aws/dms/tasks.yml

```diff
@@ -1,3 +1,25 @@
+# These configuration settings are utilized by data.yml.erb to generate configuration of the AWS Database Migration
+# Service (DMS) Tasks. Merging changes to this file does NOT update the `DATA-production` Stack that provisions
+# our AWS Redshift Cluster and the DMS Replication Tasks that export data into the analytics system. Before merging
+# a Pull Request that changes this file, an Infrastructure Engineer must execute the following steps on their workstation
+# utilizing their admin AWS credentials to validate and then deploy these changes:
+#
+#    VALIDATE:
+#
+#    code-dot-org % export AWS_PROFILE=codeorg-admin
```

Comment:
tiny nit, both here and below:

I'd recommend `$` as a more-standard alternative for this documentation. Although `%` does sometimes get used as a dividing character between path and command in the bash prompt, it's usual to do so; it took me a second to realize that was the intent.

```suggestion
#    code-dot-org$ export AWS_PROFILE=codeorg-admin
```